.github/workflows/unittestall.yml

```diff
@@ -7,12 +7,13 @@ jobs:
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@master
-    - name: Switch XCode Version
-      run: sudo xcode-select -s /Applications/Xcode_11.app
+    - uses: maxim-lobanov/setup-xcode@v1.2.1
+      with:
+        xcode-version: 12.3
     - name: Test StylableSwiftUI
       shell: bash
       run: |
         cd Example
-        set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace StylableSwiftUI.xcworkspace -scheme StylableSwiftUI-Example ONLY_ACTIVE_ARCH=NO CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 11,OS=13.0' | xcpretty --color
+        set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace StylableSwiftUI.xcworkspace -scheme StylableSwiftUI-Example ONLY_ACTIVE_ARCH=NO CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 12,OS=14.3' | xcpretty --color
         cd ..
```

Comment:
^ This file can go directly into master if it works - it's nothing to do with caching ;)

Reply:
Though, should we start . . . gulp . . . a matric to test on various iOS versions?

Reply:
I'll make a new PR with these changes

Reply:
Honestly, for changes list this just make a branch and if the CI is happy just merge to master (assuming we haven't disabled that in github :)